spec/record_level/sparse_record_spec.rb

```diff
@@ -0,0 +1,1107 @@
+require 'nokogiri'
+require 'marc'
+require 'byebug'
+require 'marc_cleanup'
+
+RSpec.describe 'sparse_record?' do
+  let(:record) { MARC::Record.new_from_hash('fields' => fields, 'leader' => leader) }
+
+  describe 'when invalid form of item' do
+    let(:leader) { '01104nam a2200289 i 4500' }
+    let(:fields) do
+      [
+        { '008' => '230414s9999    xx |||||t|||||||| ||eng||' }
+      ]
+    end
+    it { expect(MarcCleanup.sparse_record?(record)).to eq true }
+  end
+  describe 'when there is no 245 field' do
+    let(:leader) { '01104nam a2200289 i 4500' }
+    let(:fields) do
+      [
+        { '008' => '230414s9999    xx ||||| |||||||| ||eng||' }
+      ]
+    end
+    it { expect(MarcCleanup.sparse_record?(record)).to eq true }
+  end
+  describe 'monographic component part bibliographic level' do
+    let(:leader) { '01104naa a2200289 i 4500' }
+    context 'when there is a 773 field' do
```

Comment:
New blank line before a new `context` or `describe` block for better readability.

Reply:
Thank you, I will clean this up with the next PR that will finish up the sparse_record testing.